fpmsyncd/routesync.h

```diff
@@ -35,12 +41,35 @@ class RouteSync : public NetMsg
     /* Handle regular route (include VRF route) */
     void onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf);
 
+    void parse_encap(struct rtattr *tb, uint32_t &encap_value, string &rmac, uint32_t &vlan);
```

Comment:
Could you please follow the same naming convention as in the rest of file?

Reply:
Done as suggested